examples/ASViewController/Sample/Base.lproj/LaunchScreen.storyboard

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Most other examples don't use storyboards because they are largely incompatible with ASDK (at lower levels).  Ideally, remove this.